home/home.css

```diff
@@ -0,0 +1,94 @@
+@import url(../CSS/navbar.css);
+
+
+main{
+
+    text-align: center;
+     padding-top: 90px;
+    
+}
+.main-content{
+    
+    margin: auto;
+    max-width: 700px;
+    width: 80%;
+    text-align: center;
+    color: #333333;
+    font-size: 3.1rem;
+    font-weight: 900;
+
+}
+
+
+.sub-content{
+    font-size:1.2rem;
+    /*padding: 1rem; */
+    margin: auto;
+    padding:  1rem 0 1.5rem ;
+    max-width:750px ;
+    width: 90%;
+    color: #333333;
```

Comment:
you can add color variable here

Reply:
also the main of your website should be named as the index.html. I assumed the home.html is your main page

Reply:
Ok I do this.
Thank you for review.